### PR TITLE
UCT/RC/TEST: Fix connect/disconnect flow in rc mlx5 and tests for it.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1211/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1818/ for details.

**Comment by yosefe** [comment]
@alex-mikheev pls review

File: src/uct/ib/rc/accel/rc_mlx5_ep.c [hunk 1]
```diff
@@ -712,6 +712,15 @@ static UCS_CLASS_CLEANUP_FUNC(uct_rc_mlx5_ep_t)
     uct_worker_progress_unregister(iface->super.super.super.worker,
                                    uct_rc_mlx5_iface_progress, iface);
     uct_ib_mlx5_put_txwq(iface->super.super.super.worker, &self->tx.wq);
+
```
Comment (alex-mikheev) [inline src/uct/ib/rc/accel/rc_mlx5_ep.c]: please comment why this magic is needed

**Comment by MattBBaker** [comment]
This was test_time.get_time again.
@boehms can we set up POD9 to skip this test?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1213/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1820/ for details.

**Comment by alex-mikheev** [comment]
:+1:

**Pull request merged**